project/plugins.sbt

```diff
@@ -1 +1,7 @@
+resolvers += "Sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/"
+
+addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.7.0-SNAPSHOT")
```

Comment:
Others versions of plugin don't work. I have been trying with new IntelliJ 15 EAP version and the import doesn't work yet

I'm going to investigate for improve it